api/tests/security/csrf/test_csrf.py

```diff
@@ -35,6 +36,27 @@ def test_crsf_extension(app):
     assert len(csrf_cookies) > 0
     assert 'Secure; HttpOnly; Path=/; SameSite=Lax' in csrf_cookies[0]
 
+def test_crsf_extension_get_existing_csrf(app):
+    """
+    When an app is built
+        and CSRF ext is applied
+            it should expose a new endpoint /csrf
+            it should set csrf cookie
+            it should return a csrf_token in a json
```

Comment:
issue(non-blocking): please update the test description as well

Reply:
Thank you, done in the next revision!